docker-compose.yml

```diff
@@ -435,7 +434,7 @@ services:
       CONNECT_INTERNAL_VALUE_CONVERTER: "org.apache.kafka.connect.json.JsonConverter"
 
       CONNECT_REST_ADVERTISED_HOST_NAME: connect
-      CONNECT_PLUGIN_PATH: "/usr/share/java,/connect-plugins,/usr/share/confluent-hub-components"
```

Comment:
Noting this removal ... I have previously relied on this path to inject additional connector runtimes, but am prepared to try a Hub CLI and custom-image-build approach instead.

Reply:
@javabrett can you please elaborate?  Just for clarification, the entire `CONNECT_PLUGIN_PATH` parameter is not removed, just the substring to the old path with the previous IRC connector plugins.

Reply:
Unimportant - just my habit when using `cp-demo` to host an additional connector ... to demo it or some other functionality, would be to drop the connector unpacked into `/connect-plugins` on the host, then I could proceed with a connector definition.  With this path removed, I will likely change to using the local-hub-install approach, or install from hub itself, using the CLI.